config/routes.rb

```diff
@@ -0,0 +1,30 @@
+Rails.application.routes.draw do
+
+  resources :drivers
+  resources :passengers do
+    resources :trips, only: [:index, :create]
+  end
+  resources :trips
```

Comment:
You don't need all the RESTful routes for trips here since you are doing index and create nested under `passengers`.